### DISCUSSION TITLE
Project two repetition of instructions

### DISCUSSION THREAD

**Discussion by shayavh** [IBAHCM/RPiR#308] [Archive] [answered]
Hi @richardreeve @soniamitchell, in project 2, the instructions basically say to try out the new functions we created twice. Once in the original demo and once in a new demo (on the last page). Is this correct, or am I misinterpreting the text?

**Comment by soniamitchell** [comment, marked as answer]
Hi @shayavh, I don't really know what you mean, but you should only be writing one demo for each project.

**Reply by shayavh** [reply]
Hi @soniamitchell, 
I mean this part:
```
Run the three functions on the populations provided and check that your results are identical to the original functions (though the diversity of rand.pop,  quadrat.pop and quadrat10.pop will only be the same if you use the same random set of values / run them at the same time!). Remember that to run both sets of functions in the same script you will need to make sure that the functions have different names or they will overwrite one another.
``` 
and this part:
```
Write a demo that loads your package (and therefore the functions you’ve written from both Project 1 and this exercise) and shows that each set of functions generate the same results for 𝑞=0 (Species richness) and 𝑞=2 (Simpson diversity). Your demo will probably need to start with the same block of code that the last demo did, so that it generates the same populations. You can just copy it from one demo to the next (or you may come up with a cleverer way of dealing with this problem).
```
But from your answer I'm guessing for practical 2, we should write a new demo then, instead of continuing on the first one right? 

Thanks :)

**Reply by soniamitchell** [reply]
Ahh.. yea, the "Running the code" bit generally gives guidance on how to write the code, problems you might encounter, and stuff like that. Whereas the "Demo" bit describes what should be included in the demo. And yes, a new demo for each project, just like the previous practical series'. Hopefully that makes sense 👍

**Reply by shayavh** [reply]
Great thanks! :)
